src/pages/Categories/CategoriesBeta.vue

```diff
@@ -21,10 +21,10 @@
 				>
 					<main-category-tile
 						tile-size="large"
+						:category-url="category.url"
 						:category-name="category.name"
 						:category-description="category.description"
-						:image-alt="contentfulAlt"
-						:image="category.image && category.image.url ? category.image.url : contentfulSrc "
+						:image="category.image && category.image.url ? category.image.url : categoryPlaceholderImgCTF "
```

Comment:
I know this is existing code, but these `&&` usages could be consolidated slightly with optional chaining: `category.image?.url`

Reply:
Unfortunately you can't do optional chaining in the template like this, but I would prefer optional chaining in a computed property

Reply:
Ah interesting, Vue 2 doesn't support optional chaining be default. My last Vue 2 project used TS, so we probably got babel transpiling for chaining by default. Looks like Vue 3 adds support!